spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1018,21 +1020,28 @@ public String getSparkUIUrl() {
   }
 
   public void populateSparkWebUrl(InterpreterContext ctx) {
-    if (sparkUrl == null) {
-      sparkUrl = getSparkUIUrl();
-      Map<String, String> infos = new java.util.HashMap<>();
-      if (sparkUrl != null) {
-        infos.put("url", sparkUrl);
-        if (ctx != null && ctx.getClient() != null) {
-          logger.info("Sending metainfos to Zeppelin server: {}", infos.toString());
-          getZeppelinContext().setEventClient(ctx.getClient());
-          ctx.getClient().onMetaInfosReceived(infos);
-        }
+    sparkUrl = getSparkUIUrl();
+    Map<String, String> infos = new java.util.HashMap<>();
+    infos.put("url", sparkUrl);
+    java.lang.Boolean uiEnabled = java.lang.Boolean.parseBoolean(
+            property.getProperty("spark.ui.enabled", "true"));
```

Comment:
so we have trim here https://github.com/apache/zeppelin/pull/2456/files#diff-dbda0c4083ad9c59ff05f0273b5e760fR176
but not this line? I think it would be better to be consistent. I'm ok if we don't trim for other boolean property value